pages/common/git-repl.md

```diff
@@ -0,0 +1,21 @@
+# git repl
+
+> Open a Git shell-like command line.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-repl>.
+
+- Open a Git shell-like command line:
+
+`git repl`
+
+- Run a git command whilst in REPL:
+
+`{{command}}`
+
+- Run a command (not through Git) whilst in REPL:
+
+`!{{command}}`
+
+- Exit REPL:
+
+`exit|quit`
```

Comment:
I think we should choose one:
```suggestion
`exit`
```

Reply:
Sure, why not. Some people may prefer to use quit instead of exit however

Reply:
Can you give an example of someone who is bored enough to? 😆

Reply:
I am a exit user, however some people might